### PR TITLE
BAEL-3370 Guide to read command

### PR DESCRIPTION
**Pull request by srzamfir**

### CODE REVIEW

**Comment by JonCook** [comment]
retest

File: linux-bash/read/src/main/bash/read_inputs.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
```
Comment (JonCook) [inline linux-bash/read/src/main/bash/read_inputs.sh]: Could we maybe add a comment at the top here with some example usage? I tried for example but didn't see any output:
- `./read_inputs.sh default_read`

File: linux-bash/read/src/main/bash/read_inputs.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# section 2.1
+default_read() {
+    read input1 input2 input3
+    echo "[$input1] [$input2] [$input3]"
+}
+
+# section 2.2
+custom_ifs_no_array(){
```
Comment (JonCook) [inline linux-bash/read/src/main/bash/read_inputs.sh]: A minor point, let's be consistent with the spaces and parenthesis like you have been with some of the other functions. ;-)

File: linux-bash/read/src/main/bash/read_inputs.sh (hunk 1 above)
Comment (srzamfir) [reply linux-bash/read/src/main/bash/read_inputs.sh]: i did a simple menu to clarify things out ... just run ./read_inputs.sh

File: linux-bash/read/src/main/bash/read_inputs.sh (hunk 2 above)
Comment (srzamfir) [reply linux-bash/read/src/main/bash/read_inputs.sh]: done

**Pull request merged**